test/Microsoft.TestPlatform.AcceptanceTests/AcceptanceTestBase.cs

```diff
@@ -46,17 +38,16 @@ public class AcceptanceTestBase : IntegrationTestBase
     public const string CoreRunnerTargetRuntime = "";
     public const string InIsolation = "/InIsolation";
 
-    public const string NETFX452_48 = "net452;net461;net472;net48";
-    public const string NETFX451_48 = "net452;net461;net472;net48";
+    public const string NETFX462_48 = "net462;net472;net48";
     public const string NETCORE21_50 = "netcoreapp2.1;netcoreapp3.1;net5.0";
-    public const string NETFX452_NET50 = "net452;net461;net472;net48;netcoreapp2.1;netcoreapp3.1;net5.0";
-    public const string NETFX452_NET31 = "net452;net461;net472;net48;netcoreapp2.1;netcoreapp3.1";
-    public const string DEFAULT_RUNNER_NETFX = "net451";
+    public const string NETFX462_NET50 = "net462;net472;net48;netcoreapp2.1;netcoreapp3.1;net5.0";
+    public const string NETFX462_NET31 = "net462;net472;net48;netcoreapp2.1;netcoreapp3.1";
+    public const string DEFAULT_RUNNER_NETFX = "net462";
     /// <summary>
     /// Our current defaults for .NET and .NET Framework.
     /// </summary>
     public const string DEFAULT_RUNNER_NETFX_AND_NET = $"{DEFAULT_RUNNER_NETFX};netcoreapp2.1";
-    public const string DEFAULT_HOST_NETFX_AND_NET = "net451;netcoreapp2.1";
+    public const string DEFAULT_HOST_NETFX_AND_NET = "net462;netcoreapp2.1";
```

Comment:
```suggestion
    public const string DEFAULT_HOST_NETFX_AND_NET = "{Net462TargetFramework};{Core21TargetFramework}";
```

Reply:
Seems more readable as it is. The constants are just underneath each other.